src/test/scala/com/twitter/finagle/postgres/integration/IntegrationSpec.scala

```diff
@@ -411,15 +411,15 @@ class IntegrationSpec extends Spec {
       }
 
       "return correct availability information" when {
-        "client is good" in {
+        "client is good" ignore {
```

Comment:
How come these are ignored? They don't pass?

Reply:
Yes, the state management inside finagle has changed even for netty3 services

Reply:
If these APIs no longer work then I feel like we should either remove them (to lampshade the breaking change) or make them work. 

@clhodapp added this functionality, maybe he can take a look at the issue or comment whether it's still relevant in the newer finagle structure?

Reply:
@crispywalrus 

I get reliable passes on a laptop if I just update the middle test to:
```
        "client is bad" in {
          val badClient: PostgresClient = getBadClient
          badClient.isAvailable must equal(false)
          Set(Status.Busy, Status.Closed) must contain (badClient.status)
        }
```

Reply:
Same for me, got this test passing all the time. With basically the same code as @clhodapp but I took the inverse approach:
```
"client is bad" in {
          val badClient: PostgresClient = getBadClient
          badClient.isAvailable must equal(false)
          badClient.status mustNot equal(Status.Open)
        }
```